### PR TITLE
Added handler for dark mode

### PR DESCRIPTION
**Pull request by Aelof3**
- Added style block to head via javascript
- Added listener to update color scheme when changed in parent

### CODE REVIEW

File: docs/page.js [hunk 1]
```diff
@@ -23,6 +23,92 @@ if ( ! window.frameElement && window.location.protocol !== 'file:' ) {
...
+							--text-color: #bbb;
+							--secondary-text-color: #666;
+
+							--border-style: 1px solid #444;
+						}`;
+
+	head.appendChild(style);
+
+	if ( style.styleSheet ){
+		// This is required for IE8 and below.
```
Comment (looeee) [inline docs/page.js:52]: I don't think we need to support IE8.

**Comment by looeee** [comment]
Related: #19603

**Comment by Mugen87** [comment]
What is the relation between this PR and #19603?

**Comment by Aelof3** [comment]
this modifies index.html to modify the color scheme, and sends a ping to the page.js in #19603 where it updates the color scheme for the doc loaded in the viewer iframe

**Comment by Mugen87** [comment]
Does it make more sense to bundle everything into on PR?

**Comment by Aelof3** [comment]
gonna be honest, this is my first change to a repo other than my own and I have no idea how to do that

**Comment by Mugen87** [comment]
How have to created the PR? By forking the repo on you computer or via the GitHub UI?

**Comment by Aelof3** [comment]
Used the edit file button on the GitHub UI, which forked it for me I am pretty sure

**Comment by Mugen87** [comment]
It's just important that you add all changed files in the same branch. Then you can submit a single PR with all changes.

I suggest you study a bit the contribution guide and try it again:

https://github.com/mrdoob/three.js/blob/dev/.github/CONTRIBUTING.md#contribution

**Comment by Aelof3** [comment]
I see, I will close this and the other one and see what I can do

**Pull request closed without merging**